lib/__tests__/functions.test.ts

```diff
@@ -0,0 +1,159 @@
+import fs, { PathLike } from 'fs-extra';
+import findup from 'findup-sync';
+import { getCwd } from '../path';
+import { downloadGithubRepoContents } from '../github';
+import { logger } from '../logger';
+import {
+  createFunction,
+  isObjectOrFunction,
+  createEndpoint,
+  createConfig,
+} from '../cms/functions';
+
+jest.mock('fs-extra');
+jest.mock('findup-sync');
+jest.mock('../path');
+jest.mock('../github');
+
+const mockedGetCwd = getCwd as jest.MockedFunction<typeof getCwd>;
+const mockedFindup = findup as jest.MockedFunction<typeof findup>;
+const mockedFsExistsSync = fs.existsSync as jest.MockedFunction<
+  typeof fs.existsSync
+>;
+const mockedFsReadFileSync = fs.readFileSync as jest.MockedFunction<
+  typeof fs.readFileSync
+>;
+
+describe('functions', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
Nit: we have [clearMocks set to true](https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/jest.config.js#L11), so this should be redundant.

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/195/commits/e98326c0500fa5716cd9765c2558f402dabeec3d